src/ai/backend/manager/api/session.py

```diff
@@ -395,6 +395,8 @@ async def _create(request: web.Request, params: Any) -> web.Response:
 
     root_ctx: RootContext = request.app['_root.context']
     app_ctx: PrivateContext = request.app['session.context']
+    database_ptask_group: aiotools.PersistentTaskGroup = app_ctx.database_ptask_group
```

Comment:
In this case, you could omit the type annotation because `PrivateContext` has the typing information.
Also, how about just using `app_ctx.database_ptask_group` instead of assigning it to a local variable for brevity?